### PR TITLE
adding v1 of the arcgis experience builder widget demo

### PR DESCRIPTION
**Pull request by matthew-ballard**
adding v1 of the arcgis experience builder widget demo

### CODE REVIEW

**Comment by matthew-ballard** [comment]
@emma-steuer FYI!

File: Planet for Experience Builder Demo Widget/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Planet Experience Builder Demo Widget
+
+This is a sample Experience build widget that shows how to integrate imagery from Planet APIs with ArcGIS.
+
+This widget is powered by the [Sentinel Hub](https://sentinel-hub.com/) APIs and requires that you have both a Planet and Sentinel Hub account.  
+
+It that allows for a user to search for and visualize imagery stored in a Sentinel Hub collection directly from within ArcGIS Experience Builder so that Planet imagery can be used in GIS workflows without needing to download any data.
```
Comment (emma-steuer) [inline Planet for Experience Builder Demo Widget/README.md]: "It allows"

File: Planet for Experience Builder Demo Widget/widget/planet-imagery/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Planet Experience Builder Demo Widget
+
+This is a sample Experience build widget that shows how to integrate imagery from Planet APIs with ArcGIS.
+
+This widget is powered by the [Sentinel Hub](https://sentinel-hub.com/) APIs and requires that you have both a Planet and Sentinel Hub account.  
+
+It that allows for a user to search for and visualize imagery stored in a Sentinel Hub collection directly from within ArcGIS Experience Builder so that Planet imagery can be used in GIS workflows without needing to download any data.
```
Comment (emma-steuer) [inline Planet for Experience Builder Demo Widget/widget/planet-imagery/README.md]: "It allows"

**Review by emma-steuer — APPROVED** [review]
Just 2 small  comments to fix, other than that LGTM!

**Pull request merged**